MUXSDKStats/MUXSDKStats/MUXSDKPlayerBinding.m

```diff
@@ -169,8 +175,7 @@ -(NSString *)getHostName:(NSString *)urlString {
 // This matters when there are multiple AVPlayer instances running simultaneously
 //
 - (BOOL) checkIfNotificationIsRelevant:(NSNotification *)notif {
```

Comment:
We listen for a few notifications that not all relate to an `AVPlayerItem`, so this method should be named more specifically.

Reply:
Yeah that works well